lms/djangoapps/ccx/views.py

```diff
@@ -237,14 +237,16 @@ def create_ccx(request, course, ccx=None):
     # Save display name explicitly
     override_field_for_ccx(ccx, course, 'display_name', name)
 
-    # Hide anything that can show up in the schedule
-    hidden = 'visible_to_staff_only'
-    for chapter in course.get_children():
-        override_field_for_ccx(ccx, chapter, hidden, True)
-        for sequential in chapter.get_children():
-            override_field_for_ccx(ccx, sequential, hidden, True)
-            for vertical in sequential.get_children():
-                override_field_for_ccx(ccx, vertical, hidden, True)
+    # if course licensing is enabled, then all units will be shown in schedule.
+    if not run_extension_point('PCO_ENABLE_COURSE_LICENSING'):
```

Comment:
Why don't you just check the site configuration? @anfbermudezme 

You are executing a run_extension_point which checks the site configuration variable `PCO_ENABLE_COURSE_LICENSING`.

Reply:
I understand that, I didn't questions the usage of `PCO_ENABLE_COURSE_LICENSING`. However, let me re elaborate my questions:

Using only the site_configurations isn't equivalent?
```suggestion
    if not configuration_helpers.get_value('PCO_ENABLE_COURSE_LICENSING', False):
```

@anfbermudezme

Reply:
I finally understand hahah, but could you please elaborate that in the description and in comments?.
 What I understand is that calling the run_extension_points makes sure that the plugin is installed and the site is configured, because if only the site is configured, then it is not enough to guarantee that course licensing is actaully working, becuase the plugin may not be installed. That correct? 

Also, could you please update the testing instructions?, I suppose that if the site conf `PCO_ENABLE_COURSE_LICENSING` is not set, then I should expect the default behavior.
 @anfbermudezme

Reply:
I finally understand hahah, but could you please elaborate that in the description and in comments?.
What I understand is that calling the run_extension_points makes sure that the plugin is installed and the site is configured, because if only the site is configured, then it is not enough to guarantee that course licensing is actaully working, becuase the plugin may not be installed. That correct?

Also, could you please update the testing instructions?, I suppose that if the site conf PCO_ENABLE_COURSE_LICENSING is not set, then I should expect the default behavior.
@anfbermudezme